tests/test_10_tools.py

```diff
@@ -0,0 +1,38 @@
+import subprocess
+import sys
+import time
+
+import pytest
+
+from cads_adaptors.tools.url_tools import try_download
+
+
+@pytest.fixture()
+def ftp(tmp_path):
+    ftp_path = tmp_path / "ftp"
+    (ftp_path / "test").mkdir(parents=True)
```

Comment:
@EddyCMWF since you are revising `try_download`, I think there's another bug.

If I don't place the file in a folder, `try_download` fails here: https://github.com/ecmwf-projects/cads-adaptors/blob/963cb049b810ae8eff85c3387d658486fec1eee5/cads_adaptors/tools/url_tools.py#L36-L37